CHANGELOG.next.asciidoc

```diff
@@ -170,7 +170,8 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 - Allow user configuration of keep-alive behaviour for HTTPJSON and CEL inputs. {issue}33951[33951] {pull}34014[34014]
 - Add support for polling system UDP stats for UDP input metrics. {pull}34070[34070]
 - Add support for recognizing the log level in Elasticsearch JVM logs {pull}34159[34159]
-- Add `take over` mode for `filestream` for simple migration from `log` inputs {pull}34292[34292]
+- Add metrics for TCP packet processing. {pull}34333[34333]
```

Comment:
I believe this is not part of your PR :)
```suggestion
```

Changelog files have got a 99% change of getting mixed up when updating the branches :/

Reply:
Not sure what you mean, this line is not added by my PR.

Reply:
I believe it came from the merge with `main`, at least it was showing as added by your PR on my diff :man_shrugging: